js/ad_manager_vast.js

```diff
@@ -1067,6 +1067,40 @@ OO.Ads.manager(function(_, $) {
     };
 
     /**
+     * Helper function to determine if the ad is a linear ad.
+     * @private
+     * @method Vast#_hasLinearAd
+     * @param {object} ad The ad object
+     * @returns {boolean} true if the ad is a linear ad, false otherwise.
+     */
+    var _hasLinearAd = _.bind(function(ad) {
+      var linearObjectKeys = _.keys(ad.linear);
```

Comment:
so the way it originally was in our code, clickTracking and tracking will always be available in the linear object. It's not great, so I'm in the middle of changing it so I don't need to use these magic values of 1/2